scripts/webpack/dev.client.config.js

```diff
@@ -121,7 +121,10 @@ module.exports = {
       'process.env.DEBUG': JSON.stringify(process.env.DEBUG),
       'process.env.PROTOO_LISTEN_PORT': JSON.stringify(process.env.PROTOO_LISTEN_PORT || 4444),
       __SOCKET_PORT__: JSON.stringify(process.env.SOCKET_PORT),
-      __STATIC_IMAGES__: JSON.stringify(`/static/images`)
+      __STATIC_IMAGES__: JSON.stringify(`/static/images`),
+      __DD_APPLICATIONID__: JSON.stringify(process.env.DD_APPLICATIONID),
```

Comment:
rather than baking them into the build assets we probably want to have the server send these.

example:
https://github.com/ParabolInc/parabol/blob/e35ac6c6a7c9004300c7d52280a20fda49f9af85/packages/server/createSSR.ts#L10



then on the client we can do the logic in the analytics component: https://github.com/ParabolInc/parabol/blob/243af2c5105254f73a5c489530355cb7db8abd96/packages/client/components/AnalyticsPage.tsx#L32